kqueen/engines/openstack_kubespray.py

```diff
@@ -334,10 +345,15 @@ def resize(self, node_count):
         node_count = int(node_count)
         master_count = len(self.cluster.metadata["resources"]["masters"])
         new_slave_count = node_count - master_count
-        if new_slave_count < 0:
-            return False, "Node count should be at least %s" % master_count
+        if new_slave_count <= 0:
+            return False, "Node count should be at least %s" % (master_count + 1)
```

Comment:
maybe ``Node amount should not be less that Master-node count ``

Reply:
May be yes, we need a better message here

Reply:
agree, coz current message is unclear imo